scripts/docker/documentation.Dockerfile

```diff
@@ -0,0 +1,54 @@
+# This will set up an Astrobee melodic docker container using the non-NASA install instructions.
+# This image builds on top of the base melodic image building the code.
+# You must set the docker context to be the repository root directory
+
+ARG UBUNTU_VERSION=18.04
```

Comment:
Unused, could be simplified away

Reply:
it's used on the line after, we use it on other images for passing in multiple ubuntu versions (that overwrites the ARG that is set by default). I can take it out since for this we only use this version, just thought it was a nice template :)